apps/studio/components/interfaces/SignIn/SignInSSOForm.tsx

```diff
@@ -53,7 +49,6 @@ const SignInSSOForm = () => {
     })
 
     if (!error) {
-      sendEvent({ action: TelemetryActions.SIGN_IN, properties: { category: 'account' } })
```

Comment:
am removing as successful call here will redirect to /sign-in-mfa, which already sends a sign in telemetry event
https://github.com/supabase/supabase/blob/login-audit-entry/apps/studio/pages/sign-in-mfa.tsx#L49